Packs/Cyberpion/Integrations/Cyberpion/Cyberpion.py

```diff
@@ -0,0 +1,530 @@
+                results[idx]['technical_details'] = formatted_technical_details
+                results[idx]['alert_type'] = ACTION_ITEM_TYPE_NAME
+            http_responses.append(results)
+            if not http_response.get('next') or max_fetch is not None:
+                # if max fetch is not none, there's a limit (< max_page_size) so don't fetch more pages
+                break
+        demisto.info(f'finished getting action items, number of pages: {len(http_responses)}, domain={domain}')
+        final_results = []
+        for response in http_responses:
+            final_results += response
+        return final_results
+
+    def get_domain_action_items(self, domain: str,
+                                min_severity: int,
+                                alert_types: list = None,
+                                show_only_active=True
+                                ) -> Dict[str, Any]:
+        # call API
+        return {
+            "Domain": domain,
+            "Vulnerabilities": self.get_action_items(domain=domain,
+                                                     min_severity=min_severity,
+                                                     alert_types=alert_types,
+                                                     show_only_active=show_only_active,
+                                                     max_fetch=None)
+        }
+
+
+''' HELPER FUNCTIONS '''
+
+
+def convert_to_demisto_severity(severity: float) -> int:
+    """Maps Cyberpion severity to Cortex XSOAR severity
+
+    Converts the Cyberpion alert severity level (1 to 10, float) to Cortex XSOAR incident severity (1 to 4)
+    for mapping.
+
+    :type severity: ``float``
+    :param severity: severity as returned from the Cyberpion API (float)
+
+    :return: Cortex XSOAR Severity (1 to 4)
+    :rtype: ``int``
+    """
+
+    # In this case the mapping is straightforward, but more complex mappings
+    # might be required in your integration, so a dedicated function is
+    # recommended. This mapping should also be documented.
+    if 0 <= severity <= 2.5:
+        return 1
+    elif 2.6 <= severity <= 5:
+        return 2
+    elif 5.1 <= severity <= 7.5:
+        return 3
+    elif 7.6 <= severity <= 10:
+        return 4
+    raise Exception('value of severity is not between 0-10. invalid value of severity: {}'.format(severity))
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    try:
+        client.get_domain_action_items(domain='company1.com', min_severity=2)
+        time.sleep(2)
```

Comment:
We do not allow using `sleep` in the integration code as it might lead to performance issues, please remove